### PR TITLE
Add BTC node image in microVm images selector

### PR DESCRIPTION
**Pull request by samaradel**
### Description

Add a BTC node image in the ```microVm``` images selector and it deployed successfully.

### Changes

image

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2009

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/weblets/micro_vm.vue [hunk 1]
```diff
@@ -233,6 +233,11 @@ const images = [
     flist: "https://hub.grid.tf/tf-official-vms/nixos-micro-latest.flist",
     entryPoint: "/entrypoint.sh",
   },
+  {
+    name: "BTC Node Mycelium",
```
Comment (amiraabouhadid) [inline packages/playground/src/weblets/micro_vm.vue:237]: do we have to have mycelium in the name?

**Review by amiraabouhadid — COMMENTED** [review]
was able to deploy good job 
image
but what about ipv6 mentioned in issue?

**Comment by samaradel** [comment]
@AhmedHanafy725 ipv6 should be added? Is there any more info?

File: packages/playground/src/weblets/micro_vm.vue (hunk 1 above)
Comment (samaradel) [reply packages/playground/src/weblets/micro_vm.vue:237]: took it from flist name, but will confirm it

**Pull request closed without merging**